### ISSUE TITLE
[web] On forms, pressing tab should switch to the next field

### ISSUE THREAD

**Issue by mdebbar** [flutter/flutter#41396]
Related to https://github.com/flutter/flutter/issues/13264

This issue is specifically to track the work on Web to enable tabbing between form fields.

**Comment by mdebbar** [comment]
The PR that enabled focus traversal shortcuts was reverted at https://github.com/flutter/flutter/pull/41945.

As soon as the shortcuts PR re-lands, this issue should be closed.

**Comment by cbenhagen** [comment]
The PR re-landed in https://github.com/flutter/flutter/pull/42278

**Comment by andredealmei** [comment]
this bug is alive on atual master branch

**Comment by cbenhagen** [comment]
@andredealmei this seems to work fine here. Please open a new issue with the output of `flutter doctor -v` and a short example.
